### ISSUE TITLE
3ds Max export help

### ISSUE THREAD

**Issue by beau-ff** [RemiArnaud/OpenCOLLADA#1]
Hi,

I am totally new to using this plugin, and only downloaded 3ds Max 2020 yesterday. I am using it to modify and replace models in a game using textools. I have downloaded the opencollada 2020 zip, extracted the files, and (I believe) copied the colladamax dle file into the plugins folder of 3ds max. I have opened 3ds max, but I don't know how to tell if the plugin has been added successfully. When attempting to import the files to textools, I have the error message that I require opencollada in order to do so. I'm not sure what to do, I would appreciate some advice :)

**Comment by RemiArnaud** [comment]
I don't know anything about textool, but I just tested the openCOLLADA plugin for Max2020.

1) download the release from https://github.com/RemiArnaud/OpenCOLLADA/releases/tag/v1.6.70-max
  I took the Release_Max2020_static.zip

(tested also with Release_Max2020.zip)

2) read the installation guide:
  https://github.com/RemiArnaud/OpenCOLLADA/releases/download/v1.6.70-max/INSTALL_WIN.txt
  the only thing to do is:
Copy the plugin (ColladaMaxNew.dle) in the "plugins" directory of the 3dsMax 

3) open the zip, and copy the ColladaMaxNew.dle in the Max2020 plugin folder. usually located at C:\Program Files\Autodesk\3ds Max 2020\Plugins

4) open Max2020
   - test import
    File->Import->Import...
   in 'Files of Type' select OpenCOLLADA (*.DAE)
   select a .dae file, and click open to import

  - test export
    File->Export->Export
  in 'Save as type' select OpenCOLLADA (*.DAE)
  give a name to the file and click save to export

Which part does not work for you?
